charts/federator/values.yaml

```diff
@@ -0,0 +1,24 @@
+replicaCount: 3
+
+image:
+  repository: quay.io/wire/federator
+  tag: do-not-use
+
+service:
+  externalPort: 8080
+  internalPort: 8080
+
+resources:
+  # FUTUREWORK: come up with numbers which didn't appear out of thin air
```

Comment:
I think we can probably leave the resources block entirely empty: `resources: {}`.
But I'm also okay to follow the other charts and to make a change for all charts at once regarding resource usage, instead of only for the federator.

Reply:
Yeah, let's do them all together.